src/open_inwoner/search/tests/test_page.py

```diff
@@ -119,7 +119,7 @@ def test_pagination_links(self):
         self.assertEqual(response.status_code, 200)
 
         results = response.context["paginator"].object_list
-        self.assertEqual(len(results), 21)
+        self.assertEqual(len(results), 10)
```

Comment:
If the previous comment is implemented this `10` can be the RESULTS_PER_PAGE + 1
```suggestion
        self.assertEqual(len(results), settings.RESULTS_PER_PAGE + 1)
```